platforms/documentation/docs/src/docs/userguide/authoring-builds/other/custom_gradle_types.adoc

```diff
@@ -31,6 +31,7 @@ This chapter applies to the following types:
 - Objects created using `ObjectFactory.newInstance()`.
 - Objects created for a managed nested property.
 - Elements of a `NamedDomainObjectContainer`.
+- Types that have custom `dependencies` blocks
```

Comment:
🤔 It doesn't have to be named `dependencies` though?  You could name it `myPrerequisites {}` or `externalLibraries {}` or something.  The important part is that it extends `Dependencies` and uses `DependencyCollector`.  Is there a way to briefly say that here, or definitely in the section itself?

Reply:
You're right that we don't enforce a particular name, but I don't feel strongly that we should suggest that you can call it anything you like.  I think there's some advantage for following a convention and using `dependencies`.

Reply:
I agree, but I worry that it seems like this makes that name seem like a requirement.

Can you prefix it?  `otherDependencies`, `myDependencies` or something?

Reply:
Rereading this again, I think even if you use `dependencies` as the name in the example (I still think you should change it), you should definitely at least add a footnote somewhere that the name of the block doesn't matter.

Reply:
I don't think we should make the example do something that isn't justified. IOW, there's no strong reason to name the block anything other than `dependencies`. 

This is a bit of a style question. e.g., if you had a task that compiled C++ code and we were writing an example that used this task, we could name it `compileCpp` to be consistent with other compile tasks.  We _could_ name it `myCompileTask` just to demonstrate that the name could be anything, but if someone copies from this, I think we'd rather start with something vaguely consistent with other examples.

Reply:
Take another look, I added a note about the name not being important.  This still calls the overall feature "custom `dependencies` block"